<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ElecteurInterfaceClass</class>
 <widget class="QWidget" name="ElecteurInterfaceClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>817</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ElecteurInterface</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_nase">
   <property name="geometry">
    <rect>
     <x>116</x>
     <y>60</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_prenome">
   <property name="geometry">
    <rect>
     <x>116</x>
     <y>150</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_codee">
   <property name="geometry">
    <rect>
     <x>166</x>
     <y>310</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>46</x>
     <y>100</y>
     <width>55</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>556</x>
     <y>250</y>
     <width>55</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ville</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_villee">
   <property name="geometry">
    <rect>
     <x>596</x>
     <y>250</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nome">
   <property name="geometry">
    <rect>
     <x>116</x>
     <y>100</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nomruee">
   <property name="geometry">
    <rect>
     <x>296</x>
     <y>250</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_valideElecteur">
   <property name="geometry">
    <rect>
     <x>556</x>
     <y>500</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Valider</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdite">
   <property name="geometry">
    <rect>
     <x>116</x>
     <y>190</y>
     <width>110</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximumDateTime">
    <datetime>
     <hour>23</hour>
     <minute>59</minute>
     <second>59</second>
     <year>7931</year>
     <month>12</month>
     <day>20</day>
    </datetime>
   </property>
   <property name="maximumDate">
    <date>
     <year>7931</year>
     <month>12</month>
     <day>20</day>
    </date>
   </property>
   <property name="currentSection">
    <enum>QDateTimeEdit::DaySection</enum>
   </property>
   <property name="displayFormat">
    <string>dd-MM-yyyy</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>86</x>
     <y>310</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Code postal</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>416</x>
     <y>320</y>
     <width>55</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Province</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_prove">
   <property name="geometry">
    <rect>
     <x>486</x>
     <y>310</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>46</x>
     <y>150</y>
     <width>55</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Prenom</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Date de naissance</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>56</x>
     <y>60</y>
     <width>55</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>NAS</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>226</x>
     <y>250</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom de rue</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>296</x>
     <y>10</y>
     <width>951</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ajout d'un electeur</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_numruee">
   <property name="geometry">
    <rect>
     <x>116</x>
     <y>250</y>
     <width>49</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>99999</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>250</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Numero de rue</string>
   </property>
  </widget>
  <widget class="QPushButton" name="QuitterBouton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>500</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>QuitterBouton</sender>
   <signal>clicked()</signal>
   <receiver>ElecteurInterfaceClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>513</y>
    </hint>
    <hint type="destinationlabel">
     <x>408</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
